packages/utils/src/bulk-insert-patients.ts

```diff
@@ -154,6 +154,13 @@ function normalizeEmail(email: string | undefined): string | undefined {
   return trimmedEmail.toLowerCase();
 }
 
+function normalizeExternalId(id: string | undefined): string | undefined {
+  if (id == undefined) return undefined;
+  const trimmedId = id.trim();
+  if (trimmedId.length === 0) return undefined;
+  return trimmedId.toLowerCase();
```

Comment:
I don't think we should modify the external ID, case might be important.

Reply:
good point